src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -157,16 +159,17 @@ public ChassisSpeeds getTargetSpeeds(
    * @param driveMode The mode to display in SmartDashboard.
    */
   public void setDriveMode(String driveMode) {
-    SmartDashboard.putString("Drive mode", driveMode);
+    Shuffleboard.getTab("Driver").add("Drive mode", driveMode).withWidget(BuiltInWidgets.kTextView);
   }
 
   /** Runs every scheduler run. */
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
-    SmartDashboard.putNumber("FL Encoder", drive.getModules()[0].getAbsolutePosition());
-    SmartDashboard.putNumber("FR Encoder", drive.getModules()[1].getAbsolutePosition());
-    SmartDashboard.putNumber("BL Encoder", drive.getModules()[2].getAbsolutePosition());
-    SmartDashboard.putNumber("BR Encoder", drive.getModules()[3].getAbsolutePosition());
+    ShuffleboardTab debugTab = Shuffleboard.getTab("Debug");
```

Comment:
Set this up as an instance var to avoid garbage collector abuse. Also, [this needs to be a NetworkTables entry](https://docs.wpilib.org/en/stable/docs/software/dashboards/shuffleboard/layouts-with-code/sending-data.html#:~:text=If%20data%20needs%20to%20be%20updated%20(for%20example%2C%20the%20output%20of%20some%20calculation%20done%20on%20the%20robot)%2C%20call%20getEntry()%20after%20defining%20the%20value%2C%20then%20update%20it%20when%20needed%20or%20in%20a%20periodic%20function)